contentcuration/contentcuration/frontend/shared/mixins/draggable/constants.js

```diff
@@ -5,3 +5,33 @@ export const DraggableTypes = {
   COLLECTION: 'collection',
   REGION: 'region',
 };
+
+const { ITEM, COLLECTION, REGION, CONTAINER } = DraggableTypes;
+
+export const DraggableContainerTypes = { ITEM, COLLECTION, REGION, CONTAINER };
+
+export const DraggableSearchOrder = [ITEM, COLLECTION, REGION, CONTAINER];
+
+/** @see https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/dropEffect */
+export const DropEffect = {
+  COPY: 'copy',
+  MOVE: 'move',
+  NONE: 'none',
+};
+
+/** @see https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/effectAllowed */
+export const EffectAllowed = {
+  COPY: 'copy',
+  MOVE: 'move',
+  COPY_OR_MOVE: 'copyMove',
```

Comment:
In case we start adding more effects allowed, would it make sense to use a list of effects allowed rather than a constant for any combination of actions?

Reply:
These map directly to strings defined in the [Drag and Drop API](https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/effectAllowed) so actually the `copyMove` effect is actually a dedicated string for both.

Reply:
Ah I see- sounds good then!